webrtc.html

```diff
@@ -1547,7 +1548,58 @@
                         set <var>transceiver</var>'s <a>[[\CurrentDirection]]</a> slot
                         to an <code><a>RTCRtpTransceiverDirection</a></code>
                         value representing the direction of the corresponding
-                        media description.</p>
+                        <a>media description</a>.</p>
+                      </li>
+                      <li>
+                        <p>
+                          If the <a>media description</a> is indicated as using
+                          an existing <a>media transport</a> according to
+                          [[!BUNDLE]], let <var>transport</var> and
+                          <var>rtcpTransport</var> be the
+                          <code><a>RTCDtlsTransport</a></code> objects
+                          representing the RTP and RTCP components of that
```

Comment:
The use of "components" without any qualifier make me react, but I can't for my life think of a better term to use here, unless one want to expand this sentence quite a lot. I do wonder if it would be clearer to add a parenthesis after "components" saying "(as defined by ICE).

Reply:
I might say "transporting RTP and RTCP, respectively".

Reply:
It actually might be cleaner if creating a transceiver did create the underlying DtlsTransport and IceTransport objects.